_data/domains.yml

```diff
@@ -91,18 +95,21 @@ websites:
       url: https://www.domainmonster.com/
       img: domainmonster.png
       twitter: domainmonster
+      facebook: Domainmonster
```

Comment:
Is it worth adding this site's Facebook page? There has been no activity since October 2015.